app/config/set_parameters.php

```diff
@@ -56,3 +56,5 @@
     }
     $container->setParameter($key, $value);
 }
+
+$container->setParameter('cache.driver', extension_loaded('apc') ? 'apc': 'driver');
```

Comment:
shouldn't be "array" instead of "driver"?

Reply:
```php
$container->setParameter('env(CACHE_DRIVER)', extension_loaded('apc') ? 'apc': 'array');
```

Will define the default value of CACHE_DRIVER if not found.

This way, we may use any driver we want in PrestaShop Ready for instance.